xformers/components/residual.py

```diff
@@ -18,7 +18,7 @@
 from collections import namedtuple
 
 
-class LayerNormStyle(str, Enum):
+class ResidualNormStyle(str, Enum):
```

Comment:
some of the new transformer variants for vision (metaformer, efficientformer,..) alter the actual normalization, can be something else than layernorm. RMSnorm is also used in NLP